.mygcl.reference.yml

```diff
@@ -0,0 +1,37 @@
+# The golangci-lint version used to build the custom binary.
```

Comment:
I'm note sure how I feel about `mygcl` as a name for this. It's short and nice but not obvious or an obvious abbreviation, I guess because of the org I more often see `golangci` than `gcl`.

Since we use `golangci.yml` for the other configuration file, would something like `my-golangci.yml` or maybe `custom-golangci.yml` make sense? Or since it's purely for plugins, maybe `golangci-plugins.yml`?

Reply:
I didn't use the word `golangci` because I think it would be confusing with the configuration file `.golangci`.